test/common/models/Message.js

```diff
@@ -248,6 +248,81 @@ describe('The Message class', function() {
 
   });
 
+  describe('The setIsFlagged method', function() {
+
+    it('should delegate to the jmap client, passing the id and isFlagged as false if it was not provided', function(done) {
+      new jmap.Message({
+        updateMessage: function(id, option) {
+          expect(id).to.equal('id');
+          expect(option).to.deep.equal({ isFlagged: false });
+
+          done();
+        }
+      }, 'id', 'threadId', ['inbox']).setIsFlagged();
+    });
+
+    it('should delegate to the jmap client, passing the id and the given isFlagged value', function(done) {
+      new jmap.Message({
+        updateMessage: function(id, option) {
+          expect(id).to.equal('id');
+          expect(option).to.deep.equal({ isFlagged: true });
+
+          done();
+        }
+      }, 'id', 'threadId', ['inbox']).setIsFlagged(true);
+    });
+  });
```

Comment:
Add a test to check that if I pass { isFlagged: 1 } for instance, your code casts to a `boolean` correctly.